src/ngraph/pass/algebraic_simplification.cpp

```diff
@@ -543,6 +544,83 @@ static bool simplify_reduction(shared_ptr<Node> n)
     return true;
 }
 
+static bool replace_transpose_with_reshape(shared_ptr<Node> n)
+{
+    bool rc = false;
+    auto transpose = as_type_ptr<op::Transpose>(n);
+
+    if (!transpose)
```

Comment:
may have to apply style / format?

Reply:
I used it

Reply:
i mean for braces, not sure CI will pass on style

Reply:
it did, but I'll add to be consistent